EDGAR_RSS_custom.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python3
```

Comment:
Shebang is used on Unix-like systems to tell the operating system which interpreter to use to execute the script. It's recommended for scripts like this one. It's ignored on Windows machines, since Windows uses file association  when determining how to run a file (i.e. `*.py` ➡️ use a Python interpreter).